resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -30,7 +30,7 @@ private[spark] class PythonStep(
     val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
       "null"
     } else {
-      otherPyFiles.mkString(",")
+      KubernetesFileUtils.resolveSubmittedUris(otherPyFiles, filesDownloadPath).mkString(",")
```

Comment:
I'm not aware if the PythonRunner handles the downloading of remote files. However we also don't want to be downloading twice and if the init-container is already downloading the Python files, it might be worthwhile to only refer to the pre-downloaded files anyways.